browser/src/Services/Configuration/DefaultConfiguration.ts

```diff
@@ -138,6 +138,8 @@ const BaseConfiguration: IConfigurationValues = {
     "ui.animations.enabled": true,
     "ui.fontFamily": "BlinkMacSystemFont, 'Lucida Grande', 'Segoe UI', Ubuntu, Cantarell, sans-serif",
     "ui.fontSize": "13px",
+
+    "vim.colorscheme": "onedark",
```

Comment:
What do you think about also exposing this on `config.js`?

Reply:
👍